lib/streamlit/elements/dataframe_selector.py

```diff
@@ -101,9 +102,35 @@ def dataframe(
            height: 410px
 
         """
+        num_rows = None
```

Comment:
I think this code can be moved into `_arrow_dataframe`. `_legacy_dataframe` isn't supposed to support any new features, and might be removed soon anyways.

Reply:
Sure 👍 it's moved 👍